exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3 build failure

### Operating system

Windows 10

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
D:\vcpkg>vcpkg upgrade
The following packages will be rebuilt:
  * boost-stacktrace[core,windbg]:x64-windows-static@1.85.0#3
  * boost-stacktrace[core,windbg]:x86-windows-static@1.85.0#3
  * exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3
  * exiv2[bmff,core,nls,png,xmp]:x86-windows-static@0.28.3
  * libpq[core,lz4,openssl,zlib]:x64-windows-static@16.4
  * libpq[core,lz4,openssl,zlib]:x86-windows-static@16.4
  * openimageio[core,freetype,gif,libheif,libraw,opencolorio,opencv,openjpeg,webp]:x64-windows-static@2.5.14.0
  * openimageio[core,freetype,gif,libheif,libraw,opencolorio,opencv,openjpeg,webp]:x86-windows-static@2.5.14.0
Additional packages (*) will be modified to complete this operation.
If you are sure you want to rebuild the above packages, run this command with the --no-dry-run option.

D:\vcpkg>vcpkg upgrade --no-dry-run
The following packages will be rebuilt:
  * boost-stacktrace[core,windbg]:x64-windows-static@1.85.0#3
  * boost-stacktrace[core,windbg]:x86-windows-static@1.85.0#3
  * exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3
  * exiv2[bmff,core,nls,png,xmp]:x86-windows-static@0.28.3
  * libpq[core,lz4,openssl,zlib]:x64-windows-static@16.4
  * libpq[core,lz4,openssl,zlib]:x86-windows-static@16.4
  * openimageio[core,freetype,gif,libheif,libraw,opencolorio,opencv,openjpeg,webp]:x64-windows-static@2.5.14.0
  * openimageio[core,freetype,gif,libheif,libraw,opencolorio,opencv,openjpeg,webp]:x86-windows-static@2.5.14.0
```


### Failure logs

Installing 11/16 exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3...
Building exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3...
-- Downloading https://github.com/Exiv2/exiv2/archive/v0.28.3.tar.gz -> Exiv2-exiv2-v0.28.3.tar.gz...
-- Extracting source D:/vcpkg/downloads/Exiv2-exiv2-v0.28.3.tar.gz
-- Applying patch dependencies.diff
-- Using source at D:/vcpkg/buildtrees/exiv2/src/v0.28.3-747a4cd3ad.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows-static
-- Building x64-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/vcpkg/downloads/tools/cmake-3.29.2-windows/cmake-3.29.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/vcpkg/buildtrees/exiv2/x64-windows-static-dbg
    See logs for more information:
      D:\vcpkg\buildtrees\exiv2\install-x64-windows-static-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/exiv2/portfile.cmake:47 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)


error: building exiv2:x64-windows-static failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle exiv2:x64-windows-static: 27 s
Installing 12/16 exiv2[bmff,core,nls,png,xmp]:x86-windows-static@0.28.3...
Building exiv2[bmff,core,nls,png,xmp]:x86-windows-static@0.28.3...
D:\vcpkg\triplets\community\x86-windows-static.cmake: info: loaded community triplet from here. Community triplets are not built in the curated registry and are thus less likely to succeed.
-- Using cached Exiv2-exiv2-v0.28.3.tar.gz.
-- Extracting source D:/vcpkg/downloads/Exiv2-exiv2-v0.28.3.tar.gz
-- Applying patch dependencies.diff
-- Using source at D:/vcpkg/buildtrees/exiv2/src/v0.28.3-747a4cd3ad.clean
-- Found external ninja('1.11.0').
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/vcpkg/downloads/tools/cmake-3.29.2-windows/cmake-3.29.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/vcpkg/buildtrees/exiv2/x86-windows-static-dbg
    See logs for more information:
      D:\vcpkg\buildtrees\exiv2\install-x86-windows-static-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/exiv2/portfile.cmake:47 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)


error: building exiv2:x86-windows-static failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle exiv2:x86-windows-static: 15 s

### Additional context

_No response_

Package: exiv2[bmff,core,nls,png,xmp]:x86-windows-static@0.28.3 | exiv2[bmff,core,nls,png,xmp]:x64-windows-static@0.28.3

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.41.34120.0
-    vcpkg-tool version: 2024-08-01-fd884a0d390d12783076341bd43d77c3a6a15658
    vcpkg-scripts version: bb002e69d 2024-08-20 (11 hours ago)

**To Reproduce**

`vcpkg install exiv2[bmff,core,nls,png,xmp]:x86-windows-static`

**Failure logs**

```
-- Using cached Exiv2-exiv2-v0.28.3.tar.gz.
-- Extracting source D:/vcpkg/downloads/Exiv2-exiv2-v0.28.3.tar.gz
-- Applying patch dependencies.diff
-- Using source at D:/vcpkg/buildtrees/exiv2/src/v0.28.3-747a4cd3ad.clean
-- Found external ninja('1.11.0').
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/vcpkg/downloads/tools/cmake-3.29.2-windows/cmake-3.29.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/vcpkg/buildtrees/exiv2/x86-windows-static-dbg
    See logs for more information:
      D:\vcpkg\buildtrees\exiv2\install-x86-windows-static-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/exiv2/portfile.cmake:47 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)



```

<details><summary>D:\vcpkg\buildtrees\exiv2\install-x86-windows-static-dbg-out.log</summary>

```
Change Dir: 'D:/vcpkg/buildtrees/exiv2/x86-windows-static-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j9 install
[1/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\MD5.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\MD5.cpp
[2/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\UnicodeConversions.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\UnicodeConversions.cpp
[3/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\XML_Node.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\XML_Node.cpp
[4/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\ExpatAdapter.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\ExpatAdapter.cpp
FAILED: xmpsdk/CMakeFiles/exiv2-xmp.dir/src/ExpatAdapter.cpp.obj 
C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\ExpatAdapter.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\ExpatAdapter.cpp
D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\ExpatAdapter.cpp(15): fatal error C1083: Cannot open include file: 'expat.h': No such file or directory
[5/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\WXMPIterator.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\WXMPIterator.cpp
[6/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\WXMPUtils.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\WXMPUtils.cpp
[7/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\WXMPMeta.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\WXMPMeta.cpp
[8/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\ParseRDF.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\ParseRDF.cpp
[9/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\XMPCore_Impl.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\XMPCore_Impl.cpp
[10/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\XMPIterator.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\XMPIterator.cpp
[11/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\XMPMeta-GetSet.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\XMPMeta-GetSet.cpp
[12/102] C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1441~1.341\bin\Hostx64\x86\cl.exe   /TP -DBanAllEntityUsage=1 -DNOMINMAX -DXML_STATIC -ID:\vcpkg\buildtrees\exiv2\x86-windows-static-dbg -external:ID:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /MTd /Zi /Ob0 /Ox /Zo -std:c++17 /MP /utf-8 /showIncludes /Foxmpsdk\CMakeFiles\exiv2-xmp.dir\src\XMPMeta-Parse.cpp.obj /Fdxmpsdk\CMakeFiles\exiv2-xmp.dir\ /FS -c D:\vcpkg\buildtrees\exiv2\src\v0.28.3-747a4cd3ad.clean\xmpsdk\src\XMPMeta-Parse.cpp
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/exiv2/dependencies.diff
+++ b/ports/exiv2/dependencies.diff
@@ -55,3 +55,15 @@ index b0897c3..e559666 100644
    message ( "-- inih_INCLUDE_DIRS : " ${inih_INCLUDE_DIRS} )
    message ( "-- inih_LIBRARIES : " ${inih_LIBRARIES} )
    message ( "-- inih_inireader_INCLUDE_DIRS : " ${inih_inireader_INCLUDE_DIRS} )
+diff --git a/xmpsdk/CMakeLists.txt b/xmpsdk/CMakeLists.txt
+index 87c59c0..d2f23e6 100644
+--- a/xmpsdk/CMakeLists.txt
++++ b/xmpsdk/CMakeLists.txt
+@@ -31,6 +31,7 @@ target_include_directories(exiv2-xmp SYSTEM
+         ${PROJECT_SOURCE_DIR}/xmpsdk/include
+         ${EXPAT_INCLUDE_DIRS}
+ )
++target_link_libraries(exiv2-xmp PRIVATE expat::expat)
+ 
+ # Prevent a denial-service-attack related to XML entity expansion
+ # ("billion laughs attack").

--- a/ports/exiv2/vcpkg.json
+++ b/ports/exiv2/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "exiv2",
   "version": "0.28.3",
+  "port-version": 1,
   "description": "Image metadata library and tools",
   "homepage": "https://exiv2.org",
   "license": "GPL-2.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2582,7 +2582,7 @@
     },
     "exiv2": {
       "baseline": "0.28.3",
-      "port-version": 0
+      "port-version": 1
     },
     "expat": {
       "baseline": "2.6.2",

--- a/versions/e-/exiv2.json
+++ b/versions/e-/exiv2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "94c950d5e7bfffa984940dbbd278984f68181ab6",
+      "version": "0.28.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "de2ac0ef00a055d813578d9bc93e25f05decf5df",
       "version": "0.28.3",
       "port-version": 0

